fsr/fsr.ino

```diff
@@ -479,9 +483,11 @@ class SerialProcessor {
     // e.g. 3180 (fourth FSR, change threshold to 180)
     
     if (bytes_read < 2 || bytes_read > 5) { return; }
-
+    
     size_t sensor_index = buffer_[0] - '0';
-    if (sensor_index < 0 || sensor_index >= kNumSensors) { return; }
+    //this works for chars < '0' because they will
```

Comment:
Ah yeah good catch.  Can we change this comment to the follwoing:
```
// Since size_t is unsigned so we don't need to check for sensor_index < 0.
```